src/aureceiver.c

```diff
@@ -722,16 +746,19 @@ int aurecv_debug(struct re_printf *pf, const struct audio_recv *ar)
 		err |= mbuf_printf(mb, "       time = (not started)\n");
 	}
 
-	err |= re_hprintf(pf, "       player: %s,%s %s\n",
+	err |= mbuf_printf(mb, "       player: %s,%s %s\n",
```

Comment:
this looks like a separate bugfix ? perhaps a separate PR for this ?

Reply:
https://github.com/baresip/baresip/pull/2852